### ISSUE TITLE
Alignment with SWEET

### ISSUE THREAD

**Issue by lewismc** [mariutzica/Scientific-Variables-Ontology#1]
Hi @mariutzica the work which has been done here is excellent and very refreshing. It is very relevant for the kind of semantics we want to describe for our datasets and data repositories.
Over at the ESIP Semantic Technologies community we have been developing [SWEET](https://github.com/ESIPFed/sweet/blob/master/README.md) as a community project and have gotten excellent traction in the last year.
Rather than adding to SWEET all the time, one of our initiatives is to [align SWEET with other resources](https://github.com/ESIPFed/sweet/tree/master/alignments). Although there are only two official alignments so far, we have [plenty of others pending review](https://github.com/ESIPFed/sweet-tools/tree/master/sweet-alignment-manager/alignments) and even more proposed.
I would like to see if there are any alignments we could achieve between SWEET and the SVO. Is this something you would be interested in work on?
Thanks

**Comment by mariutzica** [comment]
@lewismc yes, this is a great idea. I will do a first pass alignment and will share the result. It may take a little while as I have a few other projects at the moment, but I will follow up here once I have something.

**Comment by lewismc** [comment]
@mariutzica thanks for the feedback, I just looked at the [core ontology files](https://github.com/mariutzica/Scientific-Variables-Ontology/tree/master/core%20ontology%20files) and this is no trivial task.
I'll have a pass at automating the alignments and post back here once this is done.
Thanks
